wp-content/themes/oceanwp-child/single-initiative.php

```diff
@@ -316,6 +316,32 @@
 						<?php
 						
 						comments_template();
+						?>
+						<h3>Updates</h3>
+						<br>
+						<?php
+						$initiative_updates = get_post_meta($the_post_id, 'initiative');
+						if(empty($initiative_updates) || ! is_array($initiative_updates)){
+							echo "No Updates";
+						}else{
+							?> 
+							<div class="row">
+								<div class="col-md-12 col-lg-12 col-sm-12">
+								<?php
+								foreach($initiative_updates[0] as $initiative_update){
+									?>
+									<p class="no-margin" style="font-weight:300"><?php echo date('F j, Y', strtotime($initiative_update['UpdateDate'])); ?></p>
```

Comment:
Please do not use inline styles.
In figma this `<p>` style is called `link`:
<img width="162" alt="image" src="https://user-images.githubusercontent.com/6459899/155306222-6fc603f1-68a9-4d6b-a4b0-48c421b19362.png">
Note it should have its own class that is applied on this `<p>`.